### PR TITLE
Fix #20624: Clear scrolling text cache on language change

### LINKED ISSUE #20624: Banner text is messed up if you change languages
### Operating System

Linux

### OpenRCT2 build

OpenRCT2, v0.4.5-107 (97427743a on develop, DEBUG)

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

_No response_

### Describe the issue

When I start the game in English and set it to Japanese, the banner text seems to jump between the two Languages. I've also tried to go the other way and saw the text erroneously jump around but remain in Japanese even though everything else was in English.

### Steps to reproduce

1. Open a park
2. Change the language to Japanese
3. Watch all the animated banners mess up.

### Attachments

[banners.out.webm](https://github.com/OpenRCT2/OpenRCT2/assets/7877312/10be689a-2206-4714-8301-b604daf36ee3)

### PR DESCRIPTION
**Pull request by mrmbernardi**
Fix for #20624

### CODE REVIEW

**Review by ZehMatt — APPROVED** [review]

**Comment by ZehMatt** [comment]
I think a changelog entry is in order for this one.

File: distribution/changelog.txt [hunk 1]
```diff
@@ -19,6 +19,7 @@
 - Fix: [#20778] [Plugin] Incorrect target api when executing custom actions.
 - Fix: [#20807] Tertiary colour not copied with small scenery.
 - Fix: [#20964] Crash when player connects to server with a group assigned that no longer exists.
+- Fix: [#20624] Scrolling text glitches after language is changed.
```
Comment (tupaschoal) [inline distribution/changelog.txt:22]: Broke the sorting

Comment (mrmbernardi) [reply distribution/changelog.txt:22]: Oops, my bad. I thought there was a CI check for this. Should someone fix it in the next commit?

Comment (tupaschoal) [reply distribution/changelog.txt:22]: We never merged it :'(
https://github.com/OpenRCT2/OpenRCT2/pull/20541

**Pull request merged**